client-v2/src/components/EditionFeedSectionHeader.tsx

```diff
@@ -66,6 +67,17 @@ class EditionFeedSectionHeader extends React.Component<ComponentProps> {
             </EditionDate>
           </EditionIssueInfo>
         </ManageLink>
+        &nbsp;
+        <Button
+          data-testid="check-edition-button"
+          size="s"
+          priority="primary"
+          onClick={() => this.check()}
+          tabIndex={-1}
+          title="Check Edition"
+        >
+          Check
+        </Button>
```

Comment:
it may make sense to make to button colour a bit different then publish :)

Reply:
David didn't seem bothered.  I like it as is :)